diff --git a/sites/all/modules/contrib/commerce_braintree/commerce_braintree.module b/sites/all/modules/contrib/commerce_braintree/commerce_braintree.module
index b9f06e3..c82b155 100644
--- a/sites/all/modules/contrib/commerce_braintree/commerce_braintree.module
+++ b/sites/all/modules/contrib/commerce_braintree/commerce_braintree.module
@@ -538,6 +538,11 @@ function commerce_braintree_get_feedback() {
   $feedback = FALSE;
   if (isset($_SERVER['QUERY_STRING'])) {
     $feedback = $_SERVER['QUERY_STRING'];
+
+    if (variable_get('clean_url', FALSE)) {
+      // if clean URLs enabled, remove "q" parameter from query string
+      $feedback = preg_replace("/&q(=[^&]*)?|^q(=[^&]*)?&?/", "", $feedback);
+    }
   }
   return $feedback;
 }
