diff --git a/hsts.admin.inc b/hsts.admin.inc
index 98d055e..6bfe607 100644
--- a/hsts.admin.inc
+++ b/hsts.admin.inc
@@ -51,6 +51,13 @@ function hsts_admin_settings_form() {
     '#default_value' => variable_get('hsts_subdomains', FALSE),
   );

+  $form['hsts_preload'] = array(
+    '#type' => 'checkbox',
+    '#title' => t('Preload'),
+    '#description' => t('The preload directive allows your domain to be submitted for inclusion by browsers at <a href="https://hstspreload.appspot.com/">hstspreload.appspot.com</a>. Do not enable preload unless you are sure you want all sites on your domain to be HTTPS-only for the long term.'),
+    '#default_value' => variable_get('hsts_preload', FALSE),
+  );
+
   return system_settings_form($form);
 }

diff --git a/hsts.module b/hsts.module
index cd142c7..e8be54b 100644
--- a/hsts.module
+++ b/hsts.module
@@ -27,6 +27,12 @@ function hsts_init() {
       $hsts_header .= ';includeSubDomains';
     }

+    if (variable_get('hsts_preload', FALSE)) {
+
+      // Include preload
+      $hsts_header .= ';preload';
+    }
+
     drupal_add_http_header('Strict-Transport-Security', $hsts_header);
   }
